src/main/assets/scss/main.scss

```diff
@@ -0,0 +1 @@
+@import '~govuk-frontend';
```

Comment:
Any reason you're not enabling global styles here?
`$govuk-global-styles: true`;

Reply:
https://github.com/alphagov/govuk-frontend/blob/86c85a1fa03c5130ab4132b9f3492bd47d23cbaf/docs/installation/installing-with-npm.md#global-styles

are they needed?

Reply:
Debatable I guess. I find it more convenient to have global elements pre styled. Since this repo will likely be re-used by various services, it makes more sense to let the consumer decide if it's needed. Let's go without it.